ngraph/core/include/ngraph/pattern/op/true.hpp

```diff
@@ -6,21 +6,12 @@
 
 #include "ngraph/node.hpp"
 #include "ngraph/pattern/op/pattern.hpp"
+#include "openvino/pass/pattern/op/true.hpp"
```

Comment:
do we need `ngraph/xxx` headers then?

Reply:
Yes, I don't remove these headers in order to save backward compatibility for cases when:
user includes `ngraph/xxx.hpp` header which includes `ngraph/yyy.hpp` header and user uses functionality from both headers.
If I remove these headers, it means that I can break something in user application.